pkg/cmd/pulumi/org_search.go

```diff
@@ -31,11 +32,70 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 	"github.com/spf13/cobra"
 	auto_table "go.pennock.tech/tabular/auto"
+	"go.uber.org/multierr"
+	"gopkg.in/yaml.v3"
 )
 
+type Delimiter rune
+
+func (d *Delimiter) String() string {
+	return string(*d)
+}
+
+func (d *Delimiter) Set(v string) error {
+	if v == "\\t" {
+		*d = Delimiter('\t')
+	} else if len(v) != 1 {
+		return errors.New("delimiter must be a single character")
+	} else {
+		*d = Delimiter(v[0])
+	}
+	return nil
+}
```

Comment:
This is hideous, and I'm surprised I couldn't find a better way to map characters to runes - any input would be much appreciated!

Reply:
(As it is, it allows for single-character delimiters and tabs, which should cover the vast majority of use cases, but it'd be great to be able to support more, if needed)